communities/danang.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Da Nang Agile Development Community - Da Nang",
```

Comment:
How is Da Nang related to the Software Crafters community? This looks like a Java user group. Who else in the Software Crafters Slack can confirm this entry (given that you are based in Switzerland)?

Reply:
Search for `craft` turns up 0 results: https://www.facebook.com/groups/224411477768289/search/?q=craft

Reply:
Sorry just saw your message today, I have asked the Vietnam community to update their descriptions, I come back to you asap

Reply:
Hey, is this request still up to date?

Reply:
Hi,
In theory it's been updated